apps/editing-toolkit/editing-toolkit-plugin/help-center/class-wp-rest-help-center-search.php

```diff
@@ -43,6 +43,61 @@ public function register_rest_route() {
 				),
 			)
 		);
+		register_rest_route(
+			$this->namespace,
+			'/articles',
+			array(
+				'methods'             => \WP_REST_Server::EDITABLE,
+				'callback'            => array( $this, 'get_articles_search_results' ),
```

Comment:
What do you think of moving this to `class-wp-rest-help-center-fetch-post.php`? And then have a `get_posts` method.
I think that's more appropriate, since we're not searching, but fetching posts :)

Reply:
In this case, I think we are indeed searching/querying for articles because this query is dynamic, based on path and now on tailored articles. Fetching posts happens when you click on a search result. 

Your concern brought me a question, should we consider the tailored posts when searching for a specific article? 🤔  I think we should **not consider** when searching for a specific article, WDYT?

Reply:
Yes, if the user searches for something we shouldn't consider the tailored articles!

In this case btw, I still think that the tailored articles should be considered as `fetching`. Searching is done by a query, while we just want to fetch the content of predetermined articles 😄

Reply:
In any case, if you strongly feel this is not the case, please go on with your choice!

Reply:
I have moved it